client/src/components/App/Content/InvestigationForm/useInvestigationForm.ts

```diff
@@ -119,22 +136,26 @@ const useInvestigationForm = (parameters: useInvestigationFormParameters): useIn
             showConfirmButton: false
         }
         );
-        timeout(1900).then(() => {
+        timeout(LandingPageTimer).then(() => {
             history.push(landingPageRoute);
+            interactedContactsState.interactedContacts = [];
```

Comment:
is it safe to update context state like this?

Reply:
why not?